coordinator/src/services/translation/response/tracker/translation.ts

```diff
@@ -200,29 +259,36 @@ function convertLineItemToInput(lineItemId: string, prescription: DetailPrescrip
   return taskInput
 }
 
-function convertLineItemToOutput(lineItemId: string, prescription: DetailPrescription) {
+function convertLineItemToOutput(
+  lineItemId: string,
+  prescription: spine.SummaryPrescription | spine.DetailPrescription
+) {
   const taskOutput: fhir.TaskOutput = {
     type: fhir.createCodeableConcept("http://snomed.info/sct", "373784005", "Dispensing medication"),
     valueReference: fhir.createIdentifierReference(
-      fhir.createIdentifier("https://fhir.nhs.uk/Id/prescription-dispense-item-number", lineItemId.toLowerCase()),
+      //TODO - this should be a prescription-dispense-item-number but we don't get one back in the response
+      fhir.createIdentifier("https://fhir.nhs.uk/Id/prescription-order-item-number", lineItemId.toLowerCase()),
```

Comment:
according to [IG](https://simplifier.net/guide/DigitalMedicines/NHSDigital-Task#output) it should have system `.../prescription-dispense-item-number`. this might need updating

Reply:
See the TODO above